.github/workflows/linux.yml

```diff
@@ -140,6 +140,7 @@ jobs:
           submodules: true
       - name: Install apt dependencies
         run: |
+          sudo apt-get update
```

Comment:
Isn't this needing the `\` character at the end of this line?

Reply:
I think it's fine

https://stackoverflow.com/questions/3790454/how-do-i-break-a-string-in-yaml-over-multiple-lines/21699210#21699210